staff/didac-torres/cv/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mi CV en HTML</title>
+</head>
+
+<body>
+    <header>
+        <h1>Currículum Vitae</h1>
+    </header>
+    <aside>
+        <figure>FOTOGRAFIA</figure>
```

Comment:
@didactor91 inside a figure element, you should have an img tag with its src and alt attribute to display the picture. it should contain no text.

Reply:
solved